src/screens/SSICredentialDetailsScreen/index.tsx

```diff
@@ -62,7 +63,7 @@ const SSICredentialDetailsScreen: FC<Props> = (props: Props): JSX.Element => {
         <SSITabView routes={routes} />
         {/* TODO we use this 2 button structure a lot, we should make a component out of it */}
         {(primaryAction || secondaryAction) && (
-          <ButtonContainer>
+          <ButtonContainer style={{ backgroundColor: backgrounds.secondaryDark }}>
```

Comment:
we could replace this whole view with the SSIButtonContainer component. if that is to much work, move the styling here to the correct styling component